Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -154,16 +161,37 @@ class MainTabBarViewController: UITabBarController {
         let ribbonVC = RibbonViewController(rootViewController: segmentVC)
         
         let tabRoot = HeaderViewController(rootViewController: ribbonVC)
-        return tabViewController(root: tabRoot, title: "Settings", image: UIImage(named: "tab-icon-settings")!, tag: 2)
+        return tabViewController(root: tabRoot, title: "Settings", image: UIImage(named: "tab-icon-settings")!, tag: 2, isSettingsTab: true)
     }
 
-    private func tabViewController(root: UIViewController, title: String, image: UIImage, tag: Int) -> UIViewController {
+    //TODO: Find a better way to update tab than storing a reference here
+    private var settingsTabItem : UITabBarItem? = nil
+
+    private func tabViewController(root: UIViewController, title: String, image: UIImage, tag: Int, isSettingsTab: Bool = false) -> UIViewController {
         let nav = UINavigationController(rootViewController: root)
         let tabItem = UITabBarItem(title: title, image: image, tag: tag)
+
+        //TODO: Can we not use this haclk to have a reference to the UITabBarItem?
+        if isSettingsTab {
+            settingsTabItem = tabItem
+        }
         nav.tabBarItem = tabItem
         return nav
     }
 
+    @objc func showBadge() {
+        let count = Intercom.unreadConversationCount()
+        LogService.shared.debug("showBadge() called: count: \(count), settingsTabItem:   \(settingsTabItem)")
+        if count > 0 {
+            //TODO: Find out what value just shows a badge without a number
+            settingsTabItem?.badgeValue = "\(count)"
+            //TODO: How to set to apps primary color?
+            settingsTabItem?.badgeColor = UIColor.green
```

Comment:
How should I reference the Safes primary color?

Reply:
I found `UIColor.button``
![File (2)](https://user-images.githubusercontent.com/246473/145251131-afb92e1c-ea3b-4402-b5db-40af20439258.jpg)